### ISSUE TITLE
Add instructions to send "home" command after task instructions

### ISSUE THREAD

**Issue by johngian** [satnogs/satnogs-client#53]
We need to make sure that after each observation task we send a "home" command to the ground station.

**Comment by cshields** [comment]
I don't think this is really necessary, but this could be a personal opinion..   Why put the extra wear on a rotator every time?
